tests/Http/SignupTest.php

```diff
@@ -6,10 +6,32 @@
 use Rogue\Models\Post;
 use Rogue\Models\User;
 use Rogue\Models\Signup;
+use Rogue\Services\Registrar;
 use DoSomething\Gateway\Blink;
+use DoSomething\Gateway\Resources\NorthstarUser;
 
 class SignupTest extends TestCase
 {
+    /**
+     * Helper function to mock NS API calls and receive a user.
+     */
+    public function mockNorthstarApiCallAndReturnUser($northstarId = null)
+    {
+        // Mock the Northstar API calls.
+        $this->mock(Registrar::class)
```

Comment:
@DFurnes should this also mock the `Northstar` client instead of `Registrar`?

Reply:
Ideally, yes! We always want our mocks to run as close as possible to the thing we're trying to mock (which in this case is the external HTTP call). That way, if there are bugs that pop up in the Registrar itself we'll still catch them in our unit tests.

Reply:
Makes sense, thanks for explaining, @DFurnes !